src/pocketdb/web/SearchRpc.cpp

```diff
@@ -285,6 +285,8 @@ namespace PocketWeb::PocketWebRpc
             throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, string("Invalid Pocketcoin address: ") + address);
 
         int cntOut = 10;
+        if (request.params.size() > 1 && request.params[1].isNum())
```

Comment:
Seems like we should check require.params.size() at line 279 to make sure we have arguments and determine which parameters to process.

Reply:
In fact, no - overload [] detects the presence of a field and returns NullableUniValue in its absence

Reply:
https://github.com/pocketnetteam/pocketnet.core/blob/a2475b0bdd203521b5c7f888bea00b3092d16d8c/src/univalue/lib/univalue.cpp#L197